docs/data/styles/advanced/advanced.md

```diff
@@ -2,17 +2,24 @@
 
 <p class="description">This section covers more advanced usage of @mui/styles.</p>
 
-> ⚠️ `@mui/styles` is the _**legacy**_ styling solution for Material UI.
-> It depends on [JSS](https://cssinjs.org/) as a styling solution, which is not used in the `@mui/material` anymore, deprecated in v5.
-> If you don't want to have both Emotion & JSS in your bundle, please refer to the [`@mui/system`](/system/getting-started/) documentation which is the recommended alternative.
+:::error
+`@mui/styles` was deprecated with the release of MUI Core v5 in late 2021.
```

Comment:
(Note: I was drafting this comment before it was merged 😅 Either way, I'll submit it to foster the discussion)

The only nit-pick thing — though relevant — I'd point out is the "MUI Core" usage. Given this page is sitting on the System docs, I know we're using Core here in the broad sense, but I wonder if it'd be clearer/more accurate to say "Material UI" in this case. 

The following sentence makes it slightly more tangled (as in our interchangeable use of Core/Material): "It depended on JSS..." with "it" referring "Core", right? Then, in "which is no longer used in @mui/material" we're being explicit about Material here. It sounds like we want to say "Material UI no longer uses JSS so we deprecated this library". 

Anyway, in the efforts to continue tightening up the branding stuff, this is something we may want to discuss further and resolve to avoid any more confusion in the future!

Reply:
Yeah I was torn about the usage here, but I figured "Core" was more accurate because it encompasses the suite of related packages being discussed here. Still, I can see both sides. As for "it", that was meant to refer to `@mui/styles`, so if that's unclear, it might be worthwhile to replace "it" with the actual noun it refers to.

Reply:
As an aside, I also considered mentioning `@mui/material/styles` here, because I've seen confusion between it and `@mui/styles` - they _technically_ do the same thing (apply styles to your components), and the names are almost identical, but the way they work under the hood is different, so you can't simply swap one for the other 1:1. But then I didn't want to add an additional clarification point that could just confuse some people, so I dunno. 🤷‍♂️